WordPress/gutenberg
# Add lightbox option in gallery block link control

## What?

`Expand on click` option is missing from gallery block `Link To` control. 

## Why?

Image block link control have `Expand on click` option which basically enables the lightbox on image. Gallery block link control have all other option from image block, Except `Expand on click` option.

## How?

Add `Expand on click` option in gallery block `Link To` control.

## Testing Instructions

- Make sure that you have enabled lightbox from theme.json
- Add gallery block
- Check if `Expand on click` option is available under link control
- Test `Expand on click` option function and compare it with `Image` block's lightbox functionality
- Try with different lightbox setting, and note lightbox attributes in image block under gallery block

### Testing Instructions for Keyboard


## Screenshots or screencast 
<img width="1381" alt="image" src="https://github.com/user-attachments/assets/cdce5357-ca3c-4823-940e-14c0899cd007">
